lua/grpc.lua

```diff
@@ -78,13 +78,14 @@ end
 --
 
 GRPC.methods = {}
-dofile(GRPC.basePath .. [[methods\coalitions.lua]])
 dofile(GRPC.basePath .. [[methods\atmosphere.lua]])
+dofile(GRPC.basePath .. [[methods\coalitions.lua]])
+dofile(GRPC.basePath .. [[methods\custom.lua]])
+dofile(GRPC.basePath .. [[methods\group.lua]])
+dofile(GRPC.basePath .. [[methods\mission.lua]])
 dofile(GRPC.basePath .. [[methods\trigger.lua]])
 dofile(GRPC.basePath .. [[methods\unit.lua]])
 dofile(GRPC.basePath .. [[methods\world.lua]])
-dofile(GRPC.basePath .. [[methods\custom.lua]])
-dofile(GRPC.basePath .. [[methods\event_stream.lua]])
```

Comment:
Just sorted alphabetically, because I couldn't figure out where to put my new file :D

Reply:
Good catch.